docs/anet.yml.production.template

```diff
@@ -151,253 +151,3 @@ logging:
       archivedFileCount: 2
       logFormat: '[%d{yyyy-MM-dd HH:mm:ss.SSS,UTC}] %p %c: %m%n'
 
-dictionary:
```

Comment:
you still need a
```
anetDictionaryName: "anet-dictionary.yml"
```
to be present in `anet.yml.production.template`

Reply:
it is added